about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>MWC Barcelona About me</title>
+</head>
+
+<body>
+    <header class="firstPage">
+        <nav id="firstDesktopNav">
+            <a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            <a href="https://github.com/algerina">My Page</a>
+            <a href="#">Logout</a>
+        </nav>
+        <nav class="secondDesktopNav">
+            <a href="./index.html"><img src="./img/logo4.png" alt=""></a>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="">Join</a>
+            <a href="">Sponsor</a>
+        </nav>
+        <!--mobile navbar-->
+        <section class="mobile-nav">
+            <nav id="mobileSidebar" class="sidebar">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">X</a>
+                <a href="#">About</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+                <a href="./index.html">Home</a>
+            </nav>
+            <div id="main">
+                <button class="openbtn" onclick="openNav()">☰</button>
+            </div>
+        </section>
+        <section class="sectionOneAbout">
+            <div>
+                <h1>Connectivity is unleashed at MWC Barcelona 2022</h1>
+                <p class="sectionOneAbout">MWC Barcelona attracts some of the most influential decision makers in the
+                    world.
+                    It’s where 2,400 leading companies and trailblazers share the latest topics relevant to the future.
+                    And the best place for networking opportunities with mobile and tech industry influencers.</p>
+                <h4>Please contact us per Email for any further informations
+                    about Mobile World Congress 2021! <br>
+                    mwc2021@gmail
+                </h4>
+            </div>
+        </section>
```

Comment:
- [ ] According to the design, this section does not take up the whole width. There should be equal space to the left and right of this section.
- [ ] The space between the middle text and bottom text is too much. Kindly reduce it


Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/133853526-3c7356c4-3f3c-49b3-ac05-f0daa4e70e4d.png) | ![image](https://user-images.githubusercontent.com/16058470/133853600-c841b5f0-93e9-4457-9e23-00dae392b9f6.png)